services/_shared/github_checks_client.py

```
@@ -90,4 +90,41 @@ def post_check_run(
+                headers={
+                    "Authorization": f"Bearer {install_token}",
+                    "Accept": "application/vnd.github+json",
+                    "X-GitHub-Api-Version": "2022-11-28",
+                },
+                timeout=10,
+            )
+            # Soft-fail alias: do not raise on 4xx/5xx for the mirror.
+            if alias_resp.status_code >= 400:
+                continue
+    except Exception:  # noqa: BLE001 - cutover insurance never blocks primary
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)